<ui version="4.0" >
 <class>FuelTypeDialog</class>
 <widget class="QDialog" name="FuelTypeDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Fuel Type</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="name_label" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Name:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>name_line_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="name_line_edit" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="vendor_label" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Vendor:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>vendor_line_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="vendor_line_edit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="ron_label" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>RON:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>ron_line_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="ron_line_edit" />
   </item>
   <item row="3" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_line_edit</tabstop>
  <tabstop>vendor_line_edit</tabstop>
  <tabstop>ron_line_edit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FuelTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>217</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FuelTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>217</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
